IPython/html/static/widgets/js/widget_selection.js

```diff
@@ -43,28 +44,28 @@ define([
                 this.update_button_style();
             }, this);
             this.update_button_style('');
-            
+
```

Comment:
When working on IPython, can you configure your editor not to strip trailing whitespace? It causes unnecessary git conflicts, and makes the diffs harder to review. Thanks.